CRM/Financial/DAO/FinancialType.php

```diff
@@ -287,17 +287,7 @@ public static function &export($prefix = FALSE) {
    * @return array
    */
   public static function indices($localize = TRUE) {
-    $indices = [
-      'UI_id' => [
-        'name' => 'UI_id',
-        'field' => [
-          0 => 'id',
```

Comment:
UM what?

Reply:
Yeah - so that is the duplicate index - duplicates the primary key